### DISCUSSION TITLE
Sensor value house load on Solax x3 Hybrid always positive?

### DISCUSSION THREAD

**Discussion by jensjansengraupa** [wills106/homeassistant-solax-modbus#424] [Q&A] [answered]
Hello, I have a QCells inverter and  2 older extra panels with microinverters connected to the house installation 
The sensor house load seems to always show positive values, whereas the Solax cloud also shows the negative load, due to more power from the microinverters then currently consumed.
Is there another sensor showing this figure?
I have a derived sensor which calculates the "real" house load (Solax house load + power from microinverters), but this exaggerates the consumption when house load is not negative in times of overproduction....

**Comment by wills106** [comment, marked as answer]
"House Load Alt" (disabled by default) is the most accurate house load as it doesn't compensate for losses in conversions.

"Measured Power" is the one where it goes negative when you pull from the Grid etc.

**Reply by jensjansengraupa** [reply]
Thank you!
I'll take the alt sensor, as measured power NOW shows the grid export 😎
Have a sunny day.

**Reply by Fuglebakken19** [reply]
Hi guys,

I'm also looking for that sensor, but I can't seem to find any of them. 
Any help on where to find it ?

Cheers
